src/vs/base/browser/markdownRenderer.ts

```diff
@@ -662,7 +664,15 @@ export function renderAsPlaintext(str: IMarkdownString | string, options?: {
 		value = `${value.substr(0, 100_000)}…`;
 	}
 
-	const html = marked.parse(value, { async: false, renderer: options?.includeCodeBlocksFences ? plainTextWithCodeBlocksRenderer.value : plainTextRenderer.value });
+	const renderer = createPlainTextRenderer();
+	if (options?.includeCodeBlocksFences) {
+		renderer.code = codeBlockFences;
+	}
+	if (options?.useLinkFormatter) {
```

Comment:
`useLinkFormatter` still feels like an odd name to me. Could we try doing one of these (best to worst):

1. Make whoever is generating the string generate a valid link in the first place
1. Make the new link logic the default behavior so we don't need an option. That seems reasonable if there's no `text`
1. Give the option a more clear name such as `useHrefForLinks`